mne/utils/docs.py

```diff
@@ -4532,7 +4532,7 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 theme : str | path-like
     Can be "auto", "light", or "dark" or a path-like to a
     custom stylesheet. For Dark-Mode and automatic Dark-Mode-Detection,
-    :mod:`qdarkstyle` and
+    `qdarkstyle <https://github.com/ColinDuquesnoy/QDarkStyleSheet>`__ and
```

Comment:
I can see the argument for dropping niche dependencies from intersphinx, but FWIW I also worry that hard-coded URLs like this will need to change eventually (eg. if QDarkStyleSheet repo ownership is transferred to an org or whatever).  I guess it's a question of how frequent the annoyance is likely to be.

Reply:
> FWIW I also worry that hard-coded URLs like this will need to change eventually

We have the same issue with intersphinx URLs, too -- people can change where their docs are hosted, and that does happen from time to time. At least when repos are moved, GitHub mirrors / redirects repo moves to the new location I think. So hopefully this is actually an improvement in that regard! And we have monthly `linkcheck` to check it for us which is probably good enough.